integration/experiment/uncore_frequency_sweep/uncore_frequency_sweep.py

```diff
@@ -95,7 +95,8 @@ def report_signals():
 
 
 def trace_signals():
-    return []
+    return ['MSR::UNCORE_PERF_STATUS:FREQ@package', 'MSR::UNCORE_RATIO_LIMIT:MAX_RATIO@package',
```

Comment:
Are traces enabled?

Reply:
No, not by default.  I was hoping to set the trace_signals() up for when they are enabled.